SketchUp/ui.rb

```diff
@@ -112,24 +112,24 @@ def self.create_cursor(filename, hot_x, hot_y)
   #   list = ["", "", "Male|Female"]
   #   input = UI.inputbox(prompts, defaults, list, "Tell me about yourself.")
   #
-  # @param prompts
+  # @param prompts [Array<String>]
   #   An array of prompt names appearing in the input box
   #   adjacent to input fields.
   #
-  # @param [optional] defaults
+  # @param defaults [Array<String>] (optional)
```

Comment:
Change the signature of the method to indicate optional arguments. Better for IDEs that way.